### PR TITLE
[draft-js-export-markdown] Add support for embeds

### PR DESCRIPTION
**Pull request by mxstbr**

### CODE REVIEW

File: packages/draft-js-export-markdown/src/stateToMarkdown.js [hunk 1]
```diff
@@ -253,6 +253,8 @@ class MarkupGenerator {
           let src = data.src || '';
           let alt = data.alt ? `${escapeTitle(data.alt)}` : '';
           return `![${alt}](${encodeURL(src)})`;
+        } else if (entity != null && entity.getType() === 'embed') {
```
Comment (sstur) [inline packages/draft-js-export-markdown/src/stateToMarkdown.js]: Is there a constant for 'embed' type? Or should we make one?

**Review by sstur — APPROVED** [review]
LGTM. 

Must be an issue with the prettier settings causing the formatting changes; which I think is holding up Travis.

Feel free to fix and land at will.

**Comment by mxstbr** [comment]
Done!

File: packages/draft-js-export-markdown/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "draft-js-export-markdown",
-  "version": "1.3.3",
+  "name": "draft-js-export-markdown-fork-mxstbr",
```
Comment (sstur) [inline packages/draft-js-export-markdown/package.json]: Is this intentional, or just a consequence of a git rebase?

**Comment by mxstbr** [comment]
Oh oopsie, I committed that on accident. One second...

**Comment by sstur** [comment]
Kool. Just fix up the CI issues (just lint/code formatting I think) and feel free to 🚢 it!

**Comment by mxstbr** [comment]
Done!

**Pull request merged**